.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: "Awesome Selfhosted Data"
+    url: "https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues"
```

The other option would be to change this to the choose page, which makes it more clear. Not entirely necessary though.
```suggestion
    url: "https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues/new/choose"
```

---

I thought about this as well, but only leading them to the new issue page, means they can look through them first before creating one, which might reduce possible duplicates.